src/route/index.test.ts

```diff
@@ -37,13 +37,7 @@ describe("API routes", () => {
           (server?.server?.address() as any).port
         }/api/v1/account-balances/${pubKey}?contract_ids=CCWAMYJME4H5CKG7OLXGC2T4M6FL52XCZ3OQOAV6LL3GLA4RO4WH3ASP`
       );
-      const data = await response.json();
```

Comment:
TODO: I need to rewrite these tests, the patterns changed pretty significantly on the Mercury side.